skyflow/vaultapi/insert.go

```diff
@@ -235,7 +235,7 @@ func (InsertApi *InsertApi) constructRequestBody(record common.InsertRecords, op
 		}
 
 		finalRecord["tokenization"] = options.Tokens
-		postPayload = append(postPayload, finalRecord)
```

Comment:
append() is slower and generally frowned upon when we know the size of your slice/array ahead of time.